scripts/drtprod

```diff
@@ -67,6 +67,43 @@ case $1 in
       done
     exit 0
     ;;
+  "datadog")
+    shift
+    cluster="${1}"
+
+    dd_api_key="$(gcloud --project=cockroach-drt secrets versions access latest --secret datadog-api-key)"
+    if [ -z "${dd_api_key}" ]; then
+      echo "Missing Datadog API key!"
+      exit 1
+    fi
+
+    dd_site="us5.datadoghq.com"
+
+    # Single quotes around the shell expansion cause it to be processed on the
+    # remote host instead of the local host.
+    roachprod ssh ${cluster} -- DD_INSTALL_ONLY=true bash -c '"$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"'
+
+    roachprod ssh ${cluster} -- "sudo tee /etc/datadog-agent/datadog.yaml > /dev/null << EOF
+---
+api_key: ${dd_api_key}
+site: ${dd_site}
+tags:
+- team:drt
+- env:development
```

Comment:
We currently have some things reporting as `env:dev` - should this be using the same value or does it get normalized somewhere?

Reply:
We send `env:development` everywhere else but it looks like Datadog has a default entry in their dropdown for `env:dev` and sometimes `env:dev*`. Once data is sent for other environments they appear in the dropdown too.